### DISCUSSION TITLE
question: does uBlock Origin ship with the 'uBO_extras' filters?

### DISCUSSION THREAD

**Discussion by ghost** [finnish-easylist-addition/finnish-easylist-addition#443] [Q&A] [answered]
https://github.com/finnish-easylist-addition/finnish-easylist-addition/blob/master/Finland_adb_uBO_extras.txt

The regional list 'fi: Adblock List for Finland', built-in to uBlock Origin, starts with:

```
[uBlock Origin]
! -----SPECIFIC FILTERS-----
```

...and the filters used on the secondary list seem to follow.

---

As an uBlock Origin user, do I need the additional 'extras' list? Please also consider clarifying this in the readme.

**Comment by peace2000** [comment, marked as answer]
Hi @07416 I converted this as a discussion.

The additional extras list loads automatically for uBO / Adguard users. It contains rules that are either uBO / Adguard specific or that are forbidden in Adblock Plus (anti adblock entries).

To put it simply: Those who use uBO or Adguard, get more rules and better experience with the Finnish list than Adblock Plus / Adblock users.

The readme says:

"Finnish Easylist addition. Works with all ad blockers, though uBlock Origin/AdGuard are recommended as this list contains some additional advanced rules for them."

More rules doesn't mean that it's worse. It means that more things are blocked and the overall result is better.

Did this answer your question?

**Reply by ghost** [reply]
> Did this answer your question?

Sure, no need to subscribe separately; from your reply:

> The additional extras list loads automatically for uBO / Adguard users.

**Comment by peace2000** [comment]
@07416 

I changed the readme https://github.com/finnish-easylist-addition/finnish-easylist-addition/commit/8bfbc6f8ee3bb4c27991ebb69d2806b1232ffdb3
